_ingest-pipelines/processors/sparse-encoding.md

```diff
@@ -36,13 +36,31 @@ The following table lists the required and optional parameters for the `sparse_e
 | Parameter  | Data type | Required/Optional  | Description  |
 |:---|:---|:---|:---|
 `model_id` | String | Required | The ID of the model that will be used to generate the embeddings. The model must be deployed in OpenSearch before it can be used in neural search. For more information, see [Using custom models within OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/using-ml-models/) and [Neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/).
+`prune_type` | String | Optional | The prune strategy for sparse vectors. Valid values are `max_ratio`, `alpha_mass`, `top_k`, `abs_value` and `none`. Default is `none`.
+`prune_ratio` | Float | Optional | The ratio for prune strategy. Required when `prune_type` is specified.
 `field_map` | Object | Required | Contains key-value pairs that specify the mapping of a text field to a `rank_features` field.
 `field_map.<input_field>` | String | Required | The name of the field from which to obtain text for generating vector embeddings.
 `field_map.<vector_field>`  | String | Required | The name of the vector field in which to store the generated vector embeddings.
 `description`  | String | Optional  | A brief description of the processor.  |
 `tag` | String | Optional | An identifier tag for the processor. Useful for debugging to distinguish between processors of the same type. |
 `batch_size` | Integer | Optional | Specifies the number of documents to be batched and processed each time. Default is `1`. |
 
+### Pruning sparse vectors
+
+Sparse vectors often have a long-tail distribution of token weights, with less important tokens occupying a significant amount of storage space. Pruning reduces index size by removing tokens with lower semantic importance, balancing a slight decrease in search relevance for a more compact index.
+
+The `sparse_encoding` processor can be used to prune sparse vectors by configuring the `prune_type` and `prune_ratio` parameters. The following table lists the supported prune options for the `sparse_encoding` processor. 
+
+| Prune type  | Valid prune ratio | Description  |
+|:---|:---|:---|
+`max_ratio` | Float [0, 1) | Prunes a sparse vector by keeping only elements whose values are within the `prune_ratio` of the largest value in the vector.
+abs_value | Float in (0, +∞) | Prunes a sparse vector by removing elements with values below the prune_ratio.
+alpha_mass | Float in [0, 1) | Prunes a sparse vector by keeping only elements whose cumulative sum of values is within the prune_ratio of the total sum.
+`top_k` | Integer (0, +∞) | Prunes a sparse vector by keeping only the top `prune_ratio` elements with the highest values.
```

Comment:
Can "top" be safely removed here (are "top" and "highest values" redundant)?

Reply:
It's better to remove "highest values".